README.md

```diff
@@ -1,2 +1,66 @@
-# dev-standards
-Documentation for all the development category standards followed at Osmosys
+# Osmosys Dev Standards
+
+Welcome to the Osmosys's Dev Standards Repository! This open-source project is a cornerstone of our commitment to excellence in software development. We've curated an extensive collection of coding standards, CI setup guides, and best practices spanning a multitude of programming languages. By collaborating here, you're contributing to the development community's efforts to streamline workflows, improve code quality, and promote best practices.
+
+## Table of Contents
+
+- [Osmosys Dev Standards](#osmosys-dev-standards)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
```

Comment:
May be consider adding 2 sections one for coding standards and one for CI setups and also giving list to CI setup documents.

Reply:
Updated the doc to have

1. Repo scope - this explains what all we offer right now
2. Individual details - based on scope we explains individual overview and languages covered